tests/unit/javadocs/BUILD

```diff
@@ -0,0 +1,29 @@
+load("@rules_java//java:defs.bzl", "java_library")
+load("//:defs.bzl", "java_export")
+
+# the apps-docs target previously failed with this setup.
+# now it succeeds.
+# todo: how to wrap this in a test target?
```

Comment:
Could use some guidance on this.

Reply:
Just add an `alias` to it, that will build it by default.